A typo (#13274)
mdn/content: files/en-us/web/api/web_workers_api/index.md

- {{DOMxRef("WorkerLocation")}}
  - : Defines the absolute location of the script executed by the {{domxref("Worker")}}.
- {{DOMxRef("SharedWorker")}}

>   - : Represents a specific kind of worker that can be _accessed_ from several {{glossary("browsing context", "browsing contexts")}}, be in several windows, iframes or even workers.

Reviewer: ```suggestion
  - : Represents a specific kind of worker that can be _accessed_ from several {{glossary("browsing context", "browsing contexts")}}, and be in several windows, iframes or even workers.
```

Reviewer 2: That doesn't make sense. What this paragraph wants to say (I think) is "can be accessed from several browsing contexts (i.e. windows, tabs, or even other workers)". So "being" was intentional here, although it's a terribly clumsy expression. See also the first para of the linked page:

> The SharedWorker interface represents a specific kind of worker that can be accessed from several browsing contexts, such as several windows, iframes or even workers. 

This is actually wrong though, isn't it? Because a worker isn't a browsing context. So perhaps it would be better to say:

> "can be accessed from several browsing contexts (i.e. windows, tabs, or iframes) or even other workers".

Reviewer: That last one sounds better to me.

Reviewer 2: @jpmedley , I pushed the change as suggested. If you approve, we can merge this. Thanks!

>   - : Represents a specific kind of worker that can be _accessed_ from several {{glossary("browsing context", "browsing contexts")}}, be in several windows, iframes or even workers.

Reviewer 2: ```suggestion
  - : Represents a specific kind of worker that can be accessed from several {glossary("browsing context", "browsing contexts")}} (i.e. windows, tabs, or iframes) or even other workers
```

>   - : Represents a specific kind of worker that can be accessed from several {glossary("browsing context", "browsing contexts")}} (i.e. windows, tabs, or iframes) or even other workers.

Reviewer 2: ```suggestion
  - : Represents a specific kind of worker that can be accessed from several {{glossary("browsing context", "browsing contexts")}} (i.e. windows, tabs, or iframes) or even other workers.
```
